index.js

```diff
@@ -62,6 +62,11 @@ Hypercore.prototype.add = function (opts) {
   return feed(this, null, opts)
 }
 
+Hypercore.prototype.use = function (ext) {
+  // WIP
+  return this.swarm._use(ext)
```

Comment:
mafintosh proposed a much lower level api see [here](https://github.com/mafintosh/hypercore/pull/5#issuecomment-171283614). Yeah it does accept a string now. I chose to go with mafintosh's api because i thought it was more practical